Link fixes + WR ticket (#2206)
cloudflare/cloudflare-docs: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md

# Querying Magic Transit Tunnel Health Check Results with GraphQL

> In this example, we're going to use the GraphQL Analytics API to query Magic Transit Health check results which are aggregated from individual health checks carried out by Cloudflare servers to GRE tunnels you've set up to work with Magic Transit during the [onboarding process](https://developers.cloudflare.com/magic-transit/get-started). We can query up to one week of data for dates up to three months ago.

Reviewer: ```suggestion
In this example, you are going to use the GraphQL Analytics API to query Magic Transit Health check results which are aggregated from individual health checks carried out by Cloudflare servers to GRE tunnels you have set up to work with Magic Transit during the [onboarding process](https://developers.cloudflare.com/magic-transit/get-started). You can query up to one week of data for dates up to three months ago.
```
